src/components/Buttons.js

```diff
@@ -20,7 +20,7 @@ export const PageButton = ({
             className={className}
```

Comment:
className can be not required props, and has default value `''`, in that case you can remove ternary operator